src/components/Game.js

```diff
@@ -33,7 +34,7 @@ let arr2 = [2, 3];
 let arr3 = [4, 5];
 let res = null;
 let listReq = null;
-let frequency = 5000 * 6;
+let frequency = 5000 * 1;
```

Comment:
5 sec?

Reply:
That was just for faster testing. I will change to 30 seconds and pull request again.